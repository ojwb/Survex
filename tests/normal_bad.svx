; pos=fail warn=2 error=3
*fix 1 0 0 0
1 2 2.13 012 0
; error: Tape reading may not be omitted
1 1a - 000 0
; error: Compass reading may not be omitted except on plumbed legs
2 3 1.00 - 0
; error: Character "*" not allowed in station name (use *SET NAMES to set allowed characters)
2 * 1.00 000 0
2 4 1.00 - DOWN
; warning: Compass reading given on plumbed leg
2 A 1.00 034 UP
; warning: Compass reading given on plumbed leg
1 B 1.00 000 UP
