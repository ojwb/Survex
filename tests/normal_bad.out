./normal_bad.svx:5:6: error: Tape reading may not be omitted
 1 1a - 000 0
      ^
./normal_bad.svx:7:10: error: Compass reading may not be omitted except on plumbed legs
 2 3 1.00 - 0
          ^
./normal_bad.svx:9:3: error: Character "*" not allowed in station name (use *SET NAMES to set allowed characters)
 2 * 1.00 000 0
   ^
./normal_bad.svx:12:10: warning: Compass reading given on plumbed leg
 2 A 1.00 034 UP
          ^~~
./normal_bad.svx:14:10: warning: Compass reading given on plumbed leg
 1 B 1.00 000 UP
          ^~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 5 survey stations, joined by 4 legs.
There are 0 loops.
Total length of survey legs =    5.13m (   5.13m adjusted)
Total plan length of survey legs =    2.13m
Total vertical length of survey legs =    3.00m
Vertical range = 2.00m (from b at 1.00m to 4 at -1.00m)
North-South range = 2.08m (from a at 2.08m to b at 0.00m)
East-West range = 0.44m (from a at 0.44m to b at 0.00m)

There were 2 warning(s) and 3 error(s) - no output files produced.
